app-modules/task/src/Notifications/TaskAssignedToUserNotification.php

```diff
@@ -75,13 +79,25 @@ public function toDatabase(object $notifiable): array
     {
         $url = EditTask::getUrl(['record' => $this->task]);
 
+        $student_prospect_url = ! empty($this->task->concern)
+          ? (
+              $this->task->concern instanceof Student
+            ? ViewStudent::getUrl(['record' => $this->task->concern])
+            : ViewProspect::getUrl(['record' => $this->task->concern])
+          )
+          : null;
+
         $title = str($this->task->title)->limit();
 
         $link = new HtmlString("<a href='{$url}' target='_blank' class='underline'>{$title}</a>");
 
+        $student_prospect_link = ! empty($this->task->concern) ? new HtmlString("<a href='{$student_prospect_url}' target='_blank' class='underline'>{$this->task->concern->full_name}</a>") : null;
+
+        $message = ! empty($this->task->concern) && ($this->task->concern instanceof Student || $this->task->concern instanceof Prospect) ? "You have been assigned a new Task: {$link} related to {$this->task->concern_type} {$student_prospect_link}" : "You have been assigned a new Task: {$link}";
```

Comment:
Please use a similar match statement here as what I described above, it doesn't matter if there is a little bit of duplication as it will probably make the code a little easier to read. Let me know if you have any questions there

Reply:
Done.
Sorry for variables name bug.

Reply:
Can you please refactor these lines into a single match statement for `$message`?

Reply:
Hi @danharrin 
currently its like below
<img width="442" alt="{2A028565-017C-4185-B28B-D5621259FBFB}" src="https://github.com/user-attachments/assets/3bb9cba7-935b-471f-967f-8421dc573097">
still need to convert in single line ?

Reply:
It can be a single line with the same arms as the previous match I asked you to write. It doesn't matter if there is duplication, it just needs to be clearer what each message is structured like

Reply:
Done